src/components/HelpMenu.tsx

```diff
@@ -117,6 +119,17 @@ export function HelpMenu({
             >
               Release notes
```

Comment:
No need to change it here unless you feel strongly, but now I wonder if this has any meaning in the web app.